docs/appendices/compatibility.txt

```diff
@@ -0,0 +1,234 @@
+features typical of a more strongly consistent database are not available.
+
+Because of this, some SQLAlchemy operations are not supported.
+
+Consult the following table for specifics:
+
++------------+-----------------------------+-----------------------------------+
+|  Category  | Methods                     | Notes                             |
++============+=============================+===================================+
+| `DQL`_     | - `filter()`_               | Supported.                        |
+|            | - `filter_by()`_            |                                   |
+|            | - `limit()`_                |                                   |
+|            | - `offset()`_               |                                   |
+|            | - `group_by()`_             |                                   |
+|            | - `order_by()`_             |                                   |
+|            +-----------------------------+-----------------------------------+
+|            | - `count()`_                | Partially supported.              |
+|            |                             |                                   |
+|            |                             | Consult the section on            |
+|            |                             | :ref:`aggregate functions         |
+|            |                             | <aggregate-functions>`.           |
+|            +-----------------------------+-----------------------------------+
+|            | - `join()`_, etc.           | Joins and subqueries are not      |
+|            | - `subquery()`_             | supported.                        |
++------------+-----------------------------+-----------------------------------+
+| `DML`_     | - `insert()`_               | Supported.                        |
+|            | - `from_select()`_          |                                   |
+|            | - `update()`_ (including    |                                   |
+|            |   `correlated updates`_)    |                                   |
+|            | - `delete()`_               |                                   |
++------------+-----------------------------+-----------------------------------+
+| `Session`_ | - `rollback()`_, etc.       | CrateDB does not support          |
+|            |                             | transactions, so this method and  |
+|            |                             | other methods that work with      |
+|            |                             | transactions will not do          |
+|            |                             | anything.                         |
++            +-----------------------------+-----------------------------------+
+|            | - `commit()`_               | Per the previous note, this       |
+|            |                             | method will only `flush()`_.      |
++------------+-----------------------------+-----------------------------------+
```

Comment:
@chaudum can you confirm this table is still accurate?

(assembled from multiple sources)

Reply:
`join()` and `subquery()` should be supported now. I guess we should write tests for it

Reply:
updated, and your comment ended up here:

https://trello.com/c/KyjUFI1n/2693